packages/manager/apps/cloud/client/app/dbaas/logs/detail/inputs/add/configure/logs-inputs-add-configure.controller.js

```diff
@@ -145,7 +145,7 @@ class LogsInputsAddConfigureCtrl {
       return this.$q.reject();
     }
     if (!this.flowggerForm.$dirty) {
-      return this.goToNetworkPage();
+      return this.$state.go('dbaas.logs.detail.inputs');
```

Comment:
Why not keep the function and update it to suit the needs ? It seems it will be easier to maintain

Reply:
At least change the function name and the state it goes to

Reply:
Since the third panel (network) disappear, i don't see the point to keep this function.

Reply:
It does not make sense to keep this function as is, but you could rename it to `goToInputs` and go to the state `dbaaas.logs.details.inputs`